docs/modules/usage/runtimes.md

```diff
@@ -0,0 +1,79 @@
+# Runtime Configuration
+
+A Runtime is an environment where the OpenHands agent can edit files and run
+commands.
+
+By default, OpenHands uses a Docker-based runtime, running on your local computer.
+This keeps OpenHands free and keeps your information private.
```

Comment:
What do you mean by "This keep OpenHands free"?

Reply:
Yeah good point, that's ambiguous, given that LLMs still cost money. Will work on it